content/codepipeline/_index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "CI/CD with CodePipeline"
+chapter: true
+weight: 64
+draft: true
+---
+
+# CI/CD with CodePipeline
+
```

Comment:
It would be cool to see an intro graphic on this page... maybe something similar to: https://ecsworkshop.com/images/cd-pipeline.svg

Reply:
Totally, good call. Let me get this PR through for testing on Friday and I will iterate quickly.